src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -477,14 +477,16 @@ public void ProcessSyncTree(Dictionary<GamePath, SyncNode> tree)
     /// </summary>
     private async ValueTask<Loadout.ReadOnly> ReprocessGameUpdates(Loadout.ReadOnly loadout)
     {
-        var diskState = loadout.Installation.DiskStateEntries
-            .Select(part => ((GamePath)part.Path, part.Hash));
-        
-        var suggestedVersionDefinition = _fileHashService.SuggestVersionData(loadout.InstallationInstance, diskState);
-        if (!suggestedVersionDefinition.HasValue)
+        var gameLocatorResults = loadout.InstallationInstance.Locator.Find(loadout.InstallationInstance.Game);
+
+        // NOTE(erri120): It would be very odd if we re-query the game, and it's not installed anymore
+        if (!gameLocatorResults.TryGetFirst(result => result.Store == loadout.InstallationInstance.Store, out var gameLocatorResult))
+        {
+            _logger.LogCritical("Found no installation of the game `{Store}`/`{Game}` anymore!", loadout.InstallationInstance.Store, loadout.InstallationInstance.Game.Name);
             return loadout;
-        
-        var newLocatorIds = suggestedVersionDefinition.Value.LocatorIds;
+        }
```

Comment:
This is very hypothetical, but would a user be able to install a GOG game and then uninstall GOG Galaxy?

Reply:
The app currently breaks if you manage a game and we can't find it anymore after you restart the app.